pkg/storage/local.go

```diff
@@ -1480,6 +1480,16 @@ func (is *ImageStoreLocal) checkCacheBlob(digest string) (string, error) {
 		return "", err
 	}
 
+	if _, err := os.Stat(dstRecord); err != nil {
+		if err := is.cache.DeleteBlob(digest, dstRecord); err != nil {
```

Comment:
actually this is not needed for local, because copyBlob() will exit with error later